apps/prairielearn/src/lib/config.js

```diff
@@ -36,6 +36,13 @@ const ConfigSchema = z.object({
   courseRepoDefaultBranch: z.string().default('master'),
   urlPrefix: z.string().default('/pl'),
   homeUrl: z.string().default('/'),
+  assetsPrefix: z
+    .string()
+    .default('/assets')
+    .refine((s) => s.startsWith('/'), {
+      message: 'must be an absolute path',
+    }),
```

Comment:
A nice feature of Zod I hadn't used before: custom validations!

Reply:
I love zod!